core/src/main/java/org/elasticsearch/search/aggregations/bucket/sampler/InternalSampler.java

```diff
@@ -48,11 +48,6 @@ public String getWriteableName() {
     }
 
     @Override
-    protected String getType() {
-        return "sampler";
```

Comment:
I think we did, but I just found it. The fromXContent() tests are curerently skipped if the register the parser under "sampler" but then find "unmapped_sampler" or "mapped_sampler" in the xContent